### PR TITLE
Fixing crash in `chunk_text` for empty file

### PR DESCRIPTION
**Pull request by jamesbraza**
It looks like `chunk_pdf` handles empty PDFs by raising `ImpossibleParsingError`, but `chunk_text` didn't have the same logic. So this PR ports the logic, and expands tests to cover this edge case

### CODE REVIEW

File: tests/test_agents.py [hunk 1]
```diff
@@ -38,8 +38,8 @@ async def test_get_directory_index(agent_test_settings: Settings) -> None:
         "title",
         "year",
     ], "Incorrect fields in index"
-    # paper.pdf + flag_day.html + bates.txt + obama.txt
-    assert len(await index.index_files) == 4, "Incorrect number of index files"
+    # paper.pdf + empty.txt + flag_day.html + bates.txt + obama.txt
+    assert len(await index.index_files) == 5, "Incorrect number of index files"
```
Comment (whitead) [inline tests/test_agents.py:42]: Shouldn't the empty not result in an index file?

Comment (mskarlin) [reply tests/test_agents.py:42]: I think it does but tracks it as importable for future reference.

File: paperqa/readers.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def chunk_pdf(
 
     if not parsed_text.content:
         raise ImpossibleParsingError(
-            "No text was parsed from the document: either empty or corrupted."
+            f"No text was parsed from the document {doc}, either empty or corrupted."
```
Comment (mskarlin) [inline paperqa/readers.py:51]: I think the repr for doc can be quite large, we should limit this

File: tests/test_agents.py (hunk 1 above)
Comment (whitead) [reply tests/test_agents.py:42]: interesting 👍

File: paperqa/readers.py (hunk 2 above)
Comment (jamesbraza) [reply paperqa/readers.py]: Good point, I went with just `doc.docname`

**Review by whitead — APPROVED** [review]

File: tests/test_agents.py (hunk 1 above)
Comment (jamesbraza) [reply tests/test_agents.py]: Alright @whitead I adjusted the test to account for the return of `index_files`, which is a `dict` with a value of `"ERROR"` for `empty.txt`. So now hopefully that's more intuitive
